fastlane/scripts/create-cloud-access-secrets.sh

```diff
@@ -20,14 +20,19 @@ cat > ../CryptomatorCommon/Sources/CryptomatorCommonCore/CloudAccessSecrets.swif
 import Foundation
 
 public enum CloudAccessSecrets {
+    public static let boxClientId = "${BOX_CLIENT_ID}"
+    public static let boxClientSecret = "${BOX_CLIENT_SECRET}"
+    public static let boxRedirectURIScheme = "${BOX_CLIENT_REDIRECT_URI_SCHEME}"
+    public static let boxRedirectURI = "boxsdk-${BOX_CLIENT_REDIRECT_URI_SCHEME}://boxsdkoauth2redirect"
 	public static let dropboxAppKey = "${DROPBOX_APP_KEY}"
-	public static let dropboxURLScheme = "db-${DROPBOX_APP_KEY}"
-	public static let googleDriveClientId = "${GOOGLE_DRIVE_CLIENT_ID}"
-	public static let googleDriveRedirectURLScheme = "${GOOGLE_DRIVE_REDIRECT_URL_SCHEME}"
-	public static let googleDriveRedirectURL = URL(string: "${GOOGLE_DRIVE_REDIRECT_URL_SCHEME}:/oauthredirect")
-	public static let oneDriveClientId = "${ONEDRIVE_CLIENT_ID}"
-	public static let oneDriveRedirectURIScheme = "${ONEDRIVE_REDIRECT_URI_SCHEME}"
-	public static let oneDriveRedirectURI = "${ONEDRIVE_REDIRECT_URI_SCHEME}://auth"
-	public static let pCloudAppKey = "${PCLOUD_APP_KEY}"
-}
+    public static let dropboxURLScheme = "db-${DROPBOX_APP_KEY}"
+    public static let googleDriveClientId = "${GOOGLE_DRIVE_CLIENT_ID}"
+    public static let googleDriveRedirectURLScheme = "${GOOGLE_DRIVE_REDIRECT_URL_SCHEME}"
+    public static let googleDriveRedirectURL = URL(string: "${GOOGLE_DRIVE_REDIRECT_URL_SCHEME}:/oauthredirect")
+    public static let oneDriveClientId = "${ONEDRIVE_CLIENT_ID}"
+    public static let oneDriveRedirectURIScheme = "${ONEDRIVE_REDIRECT_URI_SCHEME}"
+    public static let oneDriveRedirectURI = "${ONEDRIVE_REDIRECT_URI_SCHEME}://auth"
+    public static let pCloudAppKey = "${PCLOUD_APP_KEY}"
+
+    }
```

Comment:
Revert these changes to reduce diff noise.

Reply:
Fixed in 7758020